bookkeeper-stats-providers/twitter-ostrich-provider/pom.xml

```diff
@@ -32,6 +32,38 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.libdir>${basedir}/lib</project.libdir>
   </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
@eolivelli, It could be an other way, but I tought it is better to add checkstyle only those projects those are fixed. After every project finished, it can be moved to the root.
In this way you can't left anything unchecked.
If you add checkstyle to the root and disable it in most of the projects, it could be happen, you forget to remove the disabling config.
I mean it is a step by step change in this way, but in your way the first step would be add checkstyle to root and disable in every child pom and step by step remove them.
For me it is more transparent.